pytket/conanfile.py

```diff
@@ -32,7 +32,7 @@ def package(self):
         cmake.install()
 
     def requirements(self):
-        self.requires("tket/1.2.90@tket/stable")
```

Comment:
This change is becasue I changed a link in a code comment which apparently still requrires bumping the version